src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTests.cs

```diff
@@ -31,42 +34,18 @@ public class RazorSourceGeneratorTests
         public async Task SourceGenerator_RazorFiles_Works()
         {
             // Arrange
-            var project = CreateTestProject(new()
+            var test = new RazorTest
             {
-                ["Pages/Index.razor"] = "<h1>Hello world</h1>",
-            });
-
-            var compilation = await project.GetCompilationAsync();
-            var driver = await GetDriverAsync(project);
-
-            var result = RunGenerator(compilation!, ref driver)
-                            .VerifyPageOutput(
-@"#pragma checksum ""Pages/Index.razor"" ""{ff1816ec-aa5e-4d10-87f7-6f4963833460}"" ""6b5db227a6aa2228c777b0771108b184b1fc5df3""
```

Comment:
📝 the page output was moved to an embedded resource, which is generated by modifying the `#define` at the top of the test file and otherwise loaded from disk for the test